src/components/common/ServicesTable/ServiceTable.js

```diff
@@ -17,7 +17,7 @@ import {
   LoadingOutlined,
   DeleteOutlined,
   EditOutlined,
-  ControlOutlined,
+  // ControlOutlined,
```

Comment:
I'd say we could probably do without commented code but we may implement later then that's ok.